build.gradle.kts

```diff
@@ -91,50 +92,19 @@ dependencies {
     }
     implementation("org.apache.kafka:kafka_2.13:${Versions.kafka}", excludeLog4j)
     implementation("io.confluent:kafka-avro-serializer:${Versions.confluent}", excludeLog4j)
-    implementation("io.confluent:kafka-schema-registry:${Versions.confluent}", excludeLog4j)
     constraints {
-        implementation("org.scala-lang:scala-library") {
-            because("io.confluent:kafka-schema-registry:${Versions.confluent} -> https://www.cve.org/CVERecord?id=CVE-2022-36944")
-            version {
-                strictly(Versions.scala)
-            }
-        }
-        implementation("org.yaml:snakeyaml") {
-            because("io.confluent:kafka-schema-registry:${Versions.confluent} -> https://nvd.nist.gov/vuln/detail/CVE-2022-1471")
-            version {
-                require("2.0")
-            }
-        }
-        implementation("org.glassfish:jakarta.el") {
-            because("io.confluent:kafka-schema-registry:${Versions.confluent} -> https://advisory.checkmarx.net/advisory/vulnerability/CVE-2021-28170/")
-            version {
-                require("3.0.4")
-            }
-        }
-        implementation("com.google.protobuf:protobuf-java") {
-            because("io.confluent:kafka-schema-registry:${Versions.confluent} -> https://www.cve.org/CVERecord?id=CVE-2021-22569")
-            version {
-                require("3.21.7")
-            }
-        }
-        implementation("com.google.code.gson:gson") {
-            because("io.confluent:kafka-schema-registry:${Versions.confluent} -> https://www.cve.org/CVERecord?id=CVE-2022-25647")
-            version {
-                require("2.8.9")
-            }
-        }
-        implementation("org.json:json") {
-            because("io.confluent:kafka-schema-registry:${Versions.confluent} -> https://nvd.nist.gov/vuln/detail/CVE-2022-45688")
-            version {
-                require("20231013")
-            }
-        }
```

Comment:
Dette er den du la til på linje 70, er det ikke? Eller må `json` nå være en direkte avhengighet?

Reply:
Ja, det er mange måter å gjøre dette på. Tenkte det var like greit å legge det som en egen avhengighet (for denne pakken blir resultatet det samme).